### PR TITLE
ENH: Make headlines in 03_connectivity examples

### PR DESCRIPTION
**Pull request by KamalakerDadi**
- To make the examples visually better step by step. Also, helpful
when giving tutorials.

Reviews please ?
Thanks

### CODE REVIEW

**Comment by bthirion** [comment]
LGTM. Where can one see the rendering of these examples ?

**Comment by KamalakerDadi** [comment]
>Where can one see the rendering of these examples ?

Here it is. In CircleCI -> Artifacts ->doc->_build
https://2532-1235740-gh.circle-artifacts.com/0/home/ubuntu/nilearn/doc/_build/html/auto_examples/index.html

File: examples/03_connectivity/plot_simulated_connectome.py [hunk 1]
```diff
@@ -7,16 +7,15 @@
...
-import matplotlib.pyplot as plt
-
 
 def plot_matrix(m, ylabel=""):
     abs_max = abs(m).max()
     plt.imshow(m, cmap=plt.cm.RdBu_r, interpolation="nearest",
                vmin=-abs_max, vmax=abs_max)
 
-
+###########################################################################
```
Comment (GaelVaroquaux) [inline examples/03_connectivity/plot_simulated_connectome.py]: Actually, this example needs to run in a single cell, as it displays all the information on the same figure (see the rendering). For this specific example, we need to revert.

**Comment by GaelVaroquaux** [comment]
This is very nice. I've reviewed everything, and there is just the comment that I made that is needed to address.

**Comment by GaelVaroquaux** [comment]
Great! Merging. Thank you!

**Pull request merged**